.github/workflows/pr-comment-artifact-url.yml

```diff
@@ -19,55 +19,43 @@ jobs:
           REPO: ${{ github.event.repository.name }}
           WORKFLOW_RUN_EVENT_OBJ: ${{ toJSON(github.event.workflow_run) }}
         run: |
+          set -x
+          check_suite_id=${{ github.event.workflow_run.check_suite_id }}
+          set +x
 
           PREVIOUS_JOB_ID=$(jq -r '.id' <<< "$WORKFLOW_RUN_EVENT_OBJ")
           echo "Previous Job ID: $PREVIOUS_JOB_ID"
           echo "PREVIOUS_JOB_ID=$PREVIOUS_JOB_ID" >> "$GITHUB_ENV"
 
-          ARTIFACT_URL=$(gh api "/repos/$OWNER/$REPO/actions/artifacts" \
-            --jq ".artifacts.[] |
-            select(.workflow_run.id==${PREVIOUS_JOB_ID}) |
-            select(.expired==false) |
-            .archive_download_url")
+          HEAD_SHA=$(jq -r '.pull_requests[0].head.sha' <<< "$WORKFLOW_RUN_EVENT_OBJ")
 
-          echo "ARTIFACT URL: $ARTIFACT_URL"
-          echo "ARTIFACT_URL=$ARTIFACT_URL" >> "$GITHUB_ENV"
+          PR_COMMENT="Builds for commit $HEAD_SHA: \n"
+
+          gh api "/repos/$OWNER/$REPO/actions/artifacts" \
```

Comment:
What happens if these calls fail? Can the script deal with the situation if "artifacts.json" is empty?